bbs/admin/unix/install.sh

```diff
@@ -326,14 +331,22 @@ then
   fi
 else
   say "User ${WWIV_USER} not found; creating it."
-  useradd -g ${WWIV_GROUP} -c "WWIV BBS Service Account" -d ${WWIV_DIR} \
-	  -s /sbin/nologin -m ${WWIV_USER} >> ${LOGFILE} 2>&1
+  if [[ $OSTYPE =~ "SunOS" ]]
+  then
```

Comment:
Also, skipping the architecture check is not a good idea.  The whole reason for the check originally was to make sure we aren't trying to run mismatched code (i.e., run the precompiled binary on the wrong arch).  Just because it's SunOS is not a reason to skip it; you can't assume someone isn't going to try to run the wrong binary on the wrong platform.